auxx/src/Lang/DisplayError.hs

```diff
@@ -48,7 +48,7 @@ ppTypeError :: TypeError -> Doc
 ppTypeError TypeError{..} =
         "Couldn't match expected type" <+> (highlight $ ppTypeName teExpectedType)
     <+> "with actual value"            <+> (highlight $ show teActualValue) -- TODO: CSL-1814
-    <>  "!"
+    <+> "!"
```

Comment:
What is this change for? Why add a space before exclamation mark?

Reply:
What? Is it a joke? 🤔 
I've replaced `<>` from semigroup with `<+>` which is something similar.